docker/qemu.sh

```diff
@@ -1,7 +1,17 @@
 set -ex
 
 main() {
-    local version=4.1.0
+    # Qemu versions 3.10.0 and above break 32-bit float conversions
+    # on powerpc, powerpc64, and powerpc64le. Last known working version
+    # is 3.0.1.
+    # Upstream Issue:
+    #   https://bugs.launchpad.net/qemu/+bug/1821444
+    if [[ $1 == ppc* ]]
+    then
+        local version=3.0.1
+    else
+        local version=4.1.0
```

Comment:
Define this first as a default and only change it in the PPC case.

Reply:
Done, I've squashed it back into a single commit so I don't litter the commit history.